app/scripts/lib/indexed-db-backend.ts

```diff
@@ -0,0 +1,185 @@
+type StorageKey = {
+  name: string;
+  chainId: string;
+};
+
+type StorageBackend = {
```

Comment:
Can we use the type defined in PPOM storage rather than duplicating?

I appreciate it couples it to Blockaid but that is where the interface exists and currently only imported in the blockaid build type.